src/constants/media.ts

```diff
@@ -15,11 +22,15 @@ export const supportedMediaTypes = [IMAGE, AUDIO] as const
 
 export type SupportedMediaType = typeof supportedMediaTypes[number]
 
+export const searchTypes = [ALL_MEDIA, ...mediaTypes]
```

Comment:
`SearchType` resolves to `string` now because `as const` is missing on this array.

Reply:
I'll blame WebStorm for this one.
<img width="614" alt="Screenshot 2022-03-16 at 5 41 52 PM" src="https://user-images.githubusercontent.com/16580576/158603544-b1f6baa6-e11e-4b4d-940f-a7ae5b694103.png">

Reply:
Yeah, WebStorm's TS support is inscrutable to me! I can't tell from it's UI whether it's showing me TLS information or if it's from its own JavaScript analyzer.